op-node/rollup/types.go

```diff
@@ -122,6 +123,8 @@ type Config struct {
 
 	// UsePlasma is activated when the chain is in plasma mode.
 	UsePlasma bool `json:"use_plasma"`
+
+	VerifierConfDepth uint64 `json:"verifier_conf_depth"`
```

Comment:
An alternative could be to change the `CheckRecentTxs` bool config param to an int `CheckRecentTxsDepth` and specify the depth manually, with a default of `4` which is the current default verifier conf depth.

Reply:
Nice catch, I would like to avoid adding this parameter here. This config is what defines the L2 chain for the op-node & `verifier_conf_depth` is not a consensus parameter. We may want to expose this value a different way (maybe an extended config which is returned by the op-node RPC server).

Reply:
Thanks for confirming. I suggest in this iteration to just make the depth a config parameter with default 5. If demand comes up to read this directly from op-node, we can add an rpc to op-node to retrieve some runtime config params, separate from the rollup config.

Reply:
Updated in https://github.com/ethereum-optimism/optimism/pull/10193/commits/ccaed7bca41f6a3112d5f28b4d6decf10a3bcfa9 to remove dependency on `op-node` and instead use an int `CheckRecentTxsDepth` flag as @sebastianst suggested.

Reply:
@sebastianst Should we also add protection around the `CheckRecentTxsDepth` value so that it can't be set over a maximum limit of ~120? Since archival node support is needed to look back >128 blocks, maybe 120 is a nice safe value that should be supported any normal node

Reply:
Yes good idea, let's add a check to the batcher CLI config that this is at most 128. It doesn't make sense to use this functionality with higher values than 128, and realistic values are around 4.

Reply:
Fixed